pandas/core/base.py

```diff
@@ -946,21 +949,22 @@ def argmax(self, axis=None, skipna=True, *args, **kwargs):
         Returns
         -------
         int
-            Row position of the maximum values.
+            Row position of the {op}imum value.
 
         See Also
         --------
-        numpy.ndarray.argmax : Equivalent method for numpy arrays.
-        Series.argmin : Similar method, but returning the minimum.
+        Series.argmax : Return position of the maximum value.
+        Series.argmin : Return position of the minimum value.
```

Comment:
You're not finally using oppose. I think you can replace those two lines by this, and we don't need to repeat the same page in this case. Or you can simply remove the oppose parameter of `doc` otherwise.
```suggestion
        Series.arg{oppose} : Return position of the {oppose}imum value.
```